core/services/relay/evm/functions/contract_transmitter.go

```diff
@@ -63,6 +62,9 @@ type contractTransmitter struct {
 var _ FunctionsContractTransmitter = &contractTransmitter{}
 var _ evmRelayTypes.RouteUpdateSubscriber = &contractTransmitter{}
 
+// TODO: Is static overhead per batch enough? Make it configurable? Should we also add "before"/"after" estimates from the commitment?
+const gasOverhead = 200000
```

Comment:
The most accurate would be to use `estimatedTotalCostJuels` from the commitment. This includes both the callback gas limit + overhead directly from the contract.

This way if the contract configs change the node will be in sync.

Reply:
I wonder if we can do something simpler to overestimate and still be safe. We kinda do it now with the static 2.5M gas per tx.